Modules/sre_lib.h

```diff
@@ -806,7 +807,7 @@ SRE(match)(SRE_STATE* state, const SRE_CODE* pattern, int toplevel)
                collecting backtracking points.  for other cases,
                use the MAX_REPEAT operator */
 
-            /* <REPEAT_ONE> <skip> <1=min> <2=max> item <SUCCESS> tail */
+            /* <REPEAT_ONE> <skip> <1=min> <2=max> <3=repeat_index> item <SUCCESS> tail */
```

Comment:
Seems `repeat_index` is not used here.

Reply:
I removed `repeat_index` for these 3 OPs: `REPEAT_ONE`, `MIN_REPEAT_ONE`, `POSSESSIVE_REPEAT_ONE`.